### PR TITLE
Feature/slideout ux

### PR DESCRIPTION
**Pull request by ahulab**
# Description 

Some small changes to the slideout window to match UX

- Changed user name to user email
- added dropdown in user email section for future use
- moved logout button to bottom of slideout

### CODE REVIEW

File: src/components/view/Navigation/index.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Navigation extends React.Component {
     }
 
     handleToggle () {
-        // doesn't show nevigation unless logged in.
+        // doesn't show nvigation unless logged in.
```
Comment (evandana) [inline src/components/view/Navigation/index.js]: Spelleng!

Comment (pandathegiant) [reply src/components/view/Navigation/index.js]: LOL...  sorry for the original misspelling.

File: src/components/view/Navigation/index.js [hunk 2]
```diff
@@ -72,15 +72,25 @@ class Navigation extends React.Component {
                 <AppBar onLeftIconButtonTouchTap={this.handleToggle} title={'Welcome, ' + user.displayName}>
                 </AppBar>
                 <Drawer docked={false} width={drawerWidth} open={this.state.drawerOpen} onRequestChange={() => this.setState({drawerOpen : false})}>
-                    <div style={{backgroundColor: this.props.muiTheme.appBar.color, padding: paddingSize }}>
-                        <Avatar size={avatarSize} icon={<PersonOutlineIcon/>}/>
-                        <div style={{ color: this.props.muiTheme.appBar.textColor, paddingTop: paddingSize, paddingBottom: paddingSize }}>
-                            {'Welcome, ' + user.displayName}
+                    <div style={{display: 'flex', flexDirection: 'column', height: '100%'}}>
+                        <div style={{backgroundColor: muiTheme.appBar.color, padding: paddingSize }}>
```
Comment (evandana) [inline src/components/view/Navigation/index.js]: Base off `muiTheme.palette` instead of `muiTheme.appBar`

**Comment by ahulab** [comment]
@evandana updated, pls approve

**Pull request merged**